src/Storages/MergeTree/MergeTreeBackgroundExecutor.cpp

```diff
@@ -267,11 +269,12 @@ void MergeTreeBackgroundExecutor<Queue>::routine(TaskRuntimeDataPtr item)
     try
     {
         ALLOW_ALLOCATIONS_IN_SCOPE;
+        item->task->getQueryId();
```

Comment:
Is this missing an assignment?

```
        query_id = item->task->getQueryId();
````

Reply:
Yes, thank you
https://github.com/ClickHouse/ClickHouse/pull/52261